lib/HTTP/Action.js

```diff
@@ -14,15 +14,20 @@ class HTTPAction extends Action {
     const path = this.toPath(q);
     const { pathname, query } = url.parse(path, false, true);
     const uri = url.format(Object.assign({}, this.httpConfig, { pathname, query }));
-    return await fetch(uri, {
-      method: 'POST',
+    const request = {
```

Comment:
does ESLint say anything about the function and this const having the same name?